example.md

```diff
@@ -66,24 +61,37 @@ image: https://www.davebitter.com/img/dave.jpg
 - <SocialLink network="linkedin" handle="davebitter" />
 - <SocialLink network="twitter" handle="dave_bitter" />
 ---
-layout: intro
-image: https://avatars.githubusercontent.com/u/6022537?v=4
+layout: image-right
+image: 'https://source.unsplash.com/collection/94734566/1920x1080'
 ---
 
-# Arno Koehler
+# Code
+Section: What is Slidev?
```

Comment:
Voordat we de PR mergen deze "test" code eruit halen voor alle slides.

Reply:
You mean the indicators of which sections they're for? I'll remove them

Reply:
Yes